File: pueblo_golf_tournament_web/lib/models/team.dart
import 'package:json_annotation/json_annotation.dart';
import 'base-model.dart';
part 'team.g.dart';

@JsonSerializable()
class Team extends BaseModel {
  final String? name;
  final String? logoUrl;

  factory Team.fromJson(Map<String, dynamic> json) => _$TeamFromJson(json);

  Team(this.name, this.logoUrl, {required super.id});

  Map<String, dynamic> toJson() => _$TeamToJson(this);
}

File: pueblo-golf-tournament-api/Entities/Team.cs
using pueblo_golf_tournament_api.Base;

namespace pueblo_golf_tournament_api.Entities
{
    public class Team : BaseEntity
    {
        public string? Name { get; set; }
        public string? LogoUrl { get; set; }
        public string? DefaultBackgroundColor { get; set; }
        public long TeamCaptainId { get; set; }
    }
}